admin/assets/javascripts/session-expiry.js

```diff
@@ -35,7 +35,7 @@ $(function () {
       window.GOVUK.sessionExpiry.setCountdownText(minutesCountdown, remainingMinutes)
       window.setInterval(function () {
         window.GOVUK.sessionExpiry.setCountdownText(minutesCountdown, --remainingMinutes)
-        if (remainingMinutes === 0) window.GOVUK.sessionExpiry.redirectToLogout()
+        if (remainingMinutes <= 0) window.GOVUK.sessionExpiry.redirectToLogout()
```

Comment:
@jon-shipley Any impact on client side tests?
if not should there be any updates?